### ISSUE TITLE
Accuracy for Office-Home dataset is quite low

### ISSUE THREAD

**Issue by ltz0120** [NaJaeMin92/FixBi#11]
Hi,

Thank you for releasing the code.

I run the code on the Office-Home dataset with the default parameters, but I find the accuracy is very low. I also try to tune the parameters, but it seems does not work. Can you provide the parameters that you use for the Office-Home dataset? Thank you.

**Comment by NaJaeMin92** [comment]
Hi @ltz0120 

Here is my recipe for Office-Home dataset.
```
data_transforms = {
'train': transforms.Compose([
    transforms.Resize(256),
    transforms.RandomCrop(224),
    transforms.RandomHorizontalFlip(),
    transforms.ToTensor(),
    transforms.Normalize(
        mean=[0.485, 0.456, 0.406], 
        std=[0.229, 0.224, 0.225])
]),
'test': transforms.Compose([
    transforms.Resize(224),
    transforms.CenterCrop(224),
    transforms.ToTensor(),
    transforms.Normalize(
        mean=[0.485, 0.456, 0.406], 
        std=[0.229, 0.224, 0.225])
])}
```

I suggest you to tune the hyper-parameters such according to your baseline model.

**Comment by ltz0120** [comment]
The method works well in Office 31. For the Office-Home dataset, I tried to tune the parameters, but the results are still quite bad. Could you please share your pre-trained model for Office-Home dataset? Thank you.

**Comment by Arsiuuu** [comment]
@ltz0120  
Hi, would you like to share the pretrained models or codes : )
